packages/table/src/table.tsx

```diff
@@ -450,6 +450,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
+    public resizeRowsBasedOnColumnHeight(columnIndex: number) {
```

Comment:
That would be a bit of a mouthful. But yeah, sense. 

Does anyone have better ideas? Otherwise sure, I'll make the change.